src/databases/functions/users/user-set-karma-score.js

```diff
@@ -1,10 +1,34 @@
 const {User} = require('../../models');
+const {QueryTypes} = require('sequelize');
+const db = require('../../models/');
 
 module.exports = async (userId, combined_user_score, karma_score) => {
   if (!userId) return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function should throw an error when the `userId` is not provided instead of silently returning `null`. This will make it easier to debug issues related to missing `userId`.
```diff
-    if (!userId) return null;
+    if (!userId) throw new Error('userId is required');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->